Examples/Logging Tracer/LoggingSpan.swift

```diff
@@ -67,23 +67,23 @@ class LoggingSpan: Span {
         Logger.log("Span.addEvent(\(name))")
     }
 
-    public func addEvent(name: String, timestamp: Int64) {
+    public func addEvent(name: String, timestamp: UInt64) {
         Logger.log("Span.addEvent(\(name) timestamp:\(timestamp))")
     }
 
     public func addEvent(name: String, attributes: [String: AttributeValue]) {
         Logger.log("Span.addEvent(\(name), attributes:\(attributes) )")
     }
 
-    public func addEvent(name: String, attributes: [String: AttributeValue], timestamp: Int64) {
+    public func addEvent(name: String, attributes: [String: AttributeValue], timestamp: UInt64) {
```

Comment:
I'm curious if there is a built-in type for timestamp in swift?

Reply:
The most native way for timestamps in iOS is using TimeInterval that is basically a Double value containing number of seconds (with sub milliseconds precision). I thought about using it at the beginning of development, but initially discarded. I can reevaluate now with most of the library ready if it flows well with all the api's